src/Employer/Employer.Web/Views/Dashboard/Dashboard.cshtml

```diff
@@ -22,10 +22,14 @@
     {
         <partial name="_BlockedProviderAlert" for="BlockedProviderAlert" />
     }
+    @if (Model.ShowEmployerRevokedTransferredVacanciesAlert)
+    {
+        <partial name="_EmployerRevokedTransferredVacanciesAlert" for="EmployerRevokedTransferredVacanciesAlert" />
+    }
 
-    @if (Model.ShowTransferredVacanciesAlert)
+    @if (Model.ShowBlockedProviderTransferredVacanciesAlert)
     {
-        <partial name="_TransferredVacanciesAlert" for="TransferredVacanciesAlert" />
+        <partial name="_BlockedProviderTransferredVacanciesAlert" for="BlockedProviderTransferredVacanciesAlert" />
```

Comment:
Can you use the PartialNames constants file for referencing partial view names please?

Reply:
👍